blockchain/blockchain-wallet-v4-frontend
# Node 10 Support & Dependency Updates

## Description
- Adds support for Node 10 and updates to use corresponding internal Docker image
- All installs will now `npm ci` for ⚡️fast results
- Adds back generation of `package-lock.json` and disables generation of `yarn.lock` files leading to safer NPM installs
- Fixes blockchain-info-components NPM install issues
- Updates minor and patch versions of many dependencies
- Adds `.changelogrc` file

## Change Type
Bug Fix, CI/CD, Upgrades

## Testing Steps
Use Node 10 and run a clean install of the app.  Run various commands and ensure everything functions properly.

## PR Creator Checklist

## PR Reviewer Checklist
